playwright/_impl/_driver.py

```diff
@@ -22,9 +22,13 @@
 
 
 def compute_driver_executable() -> Path:
-    package_path = Path(inspect.getfile(playwright)).parent
-    platform = sys.platform
-    if platform == "win32":
+    if getattr(sys, "frozen", False):
```

Comment:
`hasattr` would be better
```suggestion
    if hasattr(sys, "frozen"):
```

Reply:
thats their official way how they advice it, so I would go for it since it seems tested and working.